react/package.json

```diff
@@ -17,7 +17,7 @@
   "peerDependencies": {
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
-    "@sparkdesignsystem/spark": "^12.3.0-beta.0"
+    "@sparkdesignsystem/spark": "^12.0.0"
```

Comment:
Why 12.0.0 here?

Reply:
I changed it back to ^12.0.0 cause I thought having the peer dep set to the beta was causing a problem with the build, since 12.3.0-beta.0 doesnt exist yet. Turns out it wasnt but I left it since ^12.0.0 covers up to a major anyway.